projects/dlplibs/project.yaml

```diff
@@ -1,4 +1,5 @@
 homepage: "https://www.documentliberation.org"
+main_repo: 'https://dev-www.libreoffice.org/'
```

Comment:
I did not use the link to a specific repo because the owners seem to be testing multiple repos judging from their [`Dockerfile`](https://github.com/google/oss-fuzz/blob/614e208fec139f02050594cb03c9d17a51b51ed4/projects/dlplibs/Dockerfile#L104) and [`build.sh`](https://github.com/google/oss-fuzz/blob/614e208fec139f02050594cb03c9d17a51b51ed4/projects/dlplibs/build.sh#L93).

Reply:
this needs to be a proper Git/other VCS repo, so we should use the main one, if there is one.